documentation/integration-guide.md

```diff
@@ -139,13 +139,14 @@ The JavaScript library provides utilities to start the application, send it
 commands and query its state. You can use it with any testing/assertion library
 (e.g. Jest, Mocha). The examples below are written in TypeScript using Jest.
 
-1. Add the following line to a `.npmrc` file at the root of your repository:
-    ```
-    @focusrite-novation:registry=https://npm.pkg.github.com
-    ```
+1. Install Node (you can use a Node installer, a system package manager, or a Node version manager)
```

Comment:
This could probably be fleshed out a little more, 'In your test setup' makes quite a few assumptions!